projects/experimental/components/pin/cursor/pin-cursor.component.html

```diff
@@ -0,0 +1,13 @@
+<svg
+    fill="none"
+    height="9"
+    viewBox="0 0 12 9"
+    width="12"
+    xmlns="http://www.w3.org/2000/svg"
+>
+    <!--suppress AngularUndefinedTag -->
```

Comment:
Do we need this? What if we write `<svg:path`? There has got to be a way to use SVG in Angular templates without suppressing anything :)